CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
+
+
 # 1.4.0
 
+* Dependencies needs are now being addressed with bioconda
```

Comment:
Since we've now made code changes (and will be adding fast5 support soon) should this be bumped up to release `1.5.0`?

Reply:
I decided to bump to 2.0.0 as there are some breaking changes in how the tool is installed.